docs/_data/nav.json5

```diff
@@ -44,6 +44,7 @@
             { label: 'Navigation', path: 'navigation' },
             { label: 'Pagination', path: 'pagination' },
             { label: 'Panel', path: 'panel' },
+            { label: 'Password', path: 'password' },
```

Comment:
From this https://github.com/HelixDesignSystem/helix-ui/pull/669#issuecomment-608071719:

Please update the label to `Password Input`.

Reply:
Changes have been done, please review.